.github/workflows/io_file.yml

```diff
@@ -28,25 +28,45 @@ jobs:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
       - uses: dart-lang/setup-dart@e51d8e571e22473a2ddebf0ef8a2123f0ab2c02c
         with:
-          sdk: stable
+          sdk: dev
       - run: dart pub get
       - run: dart analyze --fatal-infos
       - run: dart format --output=none --set-exit-if-changed .
 
+  check-generation:
+    # Verify that the generated code is up-to-date. Every platform/arch should
+    # generate the same API bindings.
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, ubuntu-24.04-arm, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+      - uses: dart-lang/setup-dart@e51d8e571e22473a2ddebf0ef8a2123f0ab2c02c
+        with:
+          sdk: dev
+      - run: dart pub get
+      - run: dart run ffigen --config ffigen.yaml
```

Comment:
Nit: As best practice, I've adopted having a `tool/generate.dart` that does all the generators in the project with a `--set-exit-if-changed` param. That way you can run all of them locally easily as well pre-commit.

Reply:
I'll fix this in another PR